dbusmock/pytest_fixtures.py

```diff
@@ -0,0 +1,41 @@
+'''pytest fixtures for DBusMock'''
+
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 3 of the License, or (at your option) any
+# later version.  See http://www.gnu.org/copyleft/lgpl.html for the full text
+# of the license.
+
+__author__ = 'Martin Pitt'
+__copyright__ = '(c) 2023 Martin Pitt <martin@piware.de>'
+
+from typing import Iterator
+
+import pytest
+
+import dbusmock.testcase
+
+
+@pytest.fixture(name='dbusmock_testcase', scope='session')
+def fixture_dbusmock_testcase() -> Iterator[dbusmock.testcase.DBusTestCase]:
+    '''Export the whole DBusTestCase as a fixture.'''
+
+    testcase = dbusmock.testcase.DBusTestCase()
+    yield testcase
+    testcase.tearDownClass()
+
+
+@pytest.fixture(scope='session')
+def dbusmock_testcase_system(dbusmock_testcase) -> Iterator[dbusmock.testcase.DBusTestCase]:
+    '''Export the whole DBusTestCase as a fixture, with system bus started'''
+
+    dbusmock_testcase.start_system_bus()
+    yield dbusmock_testcase
```

Comment:
need to learn/test: maybe `return` here?

Reply:
yes, `yield` is only useful if you want to do something afterwards, otherwise a return is good enough (which will also drop the `Iterator` from the type annotation.

Reply:
Fixed.